packages/backend/src/index.ts

```diff
@@ -122,11 +121,6 @@ app.use((error: Error, req: Request, res: Response, next: NextFunction) => {
     });
 });
 
-// Update all resources on startup
```

Comment:
No more update on startup (since project is no longer guaranteed to be available)